tokio/tests/task_join_set.rs

```diff
@@ -156,6 +156,45 @@ fn runtime_gone() {
         .is_cancelled());
 }
 
+#[tokio::test]
+async fn join_all() {
+    let mut set: JoinSet<i32> = JoinSet::new();
+
+    for _ in 0..5 {
+        set.spawn(async { 1 });
+    }
+    let res: Vec<i32> = set.join_all().await;
+
+    assert_eq!(res.len(), 5);
+    for itm in res.into_iter() {
+        assert_eq!(itm, 1)
+    }
+}
+
+#[tokio::test]
+async fn task_panics() {
```

Comment:
This test is failing under the `panic=abort` condition, so you could use `#[cfg(panic = "unwind")]` gate to avoid this.

```suggestion
#[cfg(panic = "unwind")]
#[tokio::test]
async fn task_panics() {
```